path_static_roles.go

```diff
@@ -77,10 +81,6 @@ func fieldsForType(roleType string) map[string]*framework.FieldSchema {
 			Type:        framework.TypeString,
 			Description: "The distinguished name of the entry to manage.",
 		},
-		"ttl": {
-			Type:        framework.TypeDurationSecond,
-			Description: "The time-to-live for the password.",
-		},
```

Comment:
I'm trying to understand why ttl is removed and the static creds seem to be based off of the RotationPeriod field, while dynamic creds are instead still using ttl? Perhaps I'm missing something bigger picture.

Reply:
If you look at the usage of `fieldsForType`, it is only called from `pathStaticRoles`. Since static roles don't use the `ttl` field, I chose to remove it from this list. Dynamic creds use the [default_ttl and max_ttl fields](https://github.com/hashicorp/vault-plugin-secrets-openldap/blob/main/path_dynamic_roles.go#L54-L61).

Reply:
I had actually removed `fieldsForType` earlier in this PR (see my comment in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/40#discussion_r960103063) because it doesn't provide much value.

Reply:
Ahh I see. I hadn't actually caught on yet that static roles don't have TTLs in general. I figured it only applied to the password in the case of static roles but rotation already covers that. That's good to have learned!